pyeudiw/satosa/default/response_handler.py

```diff
@@ -312,7 +307,8 @@ def _parse_authorization_response(self, context: Context) -> AuthorizeResponsePa
                 parser = DirectPostParser()
                 return parser.parse_and_validate(context)
             case ResponseMode.direct_post_jwt:
-                parser = DirectPostJwtJweParser(self.config["metadata_jwks"])
+                jwe_decrypter = JWEHelper(self.config["metadata_jwks"])
```

Comment:
direct_post.jwt not necessarly must be encrypted.

implementations that forces direct_post.jwt should handle this in their configuration, anyway, it would somenting mandated by wallet solutions while RP should handle interop as much as possible.

Therefore, here, we need to inspect if the jwt is encrypted or not, and handle it properly within these two posibbilities.